mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,640 @@
+                 ColumnAggregate rule which evaluates whether the column aggregate statistic lies between a specified range.
+               properties:
+                 - !ruby/object:Api::Type::Enum
+                   name: 'statistic'
+                   values:
+                     - :STATISTIC_UNDEFINED
+                     - :MEAN
+                     - :MIN
+                     - :MAX
+                 - !ruby/object:Api::Type::String
+                   name: 'minValue'
+                   description: |
+                     The minimum column statistic value allowed for a row to pass this validation.
+                     At least one of minValue and maxValue need to be provided.
+                 - !ruby/object:Api::Type::String
+                   name: maxValue
+                   description: |
+                     The maximum column statistic value allowed for a row to pass this validation.
+                     At least one of minValue and maxValue need to be provided.
+                 - !ruby/object:Api::Type::Boolean
+                   name: 'strictMinEnabled'
+                   default_value: false
+                   description: |
+                     Whether column statistic needs to be strictly greater than ('>') the minimum, or if equality is allowed.
+                     Only relevant if a minValue has been defined. Default = false.
+                 - !ruby/object:Api::Type::Boolean
+                   name: 'strictMaxEnabled'
+                   default_value: false
+                   description: |
+                     Whether column statistic needs to be strictly lesser than ('<') the maximum, or if equality is allowed.
+                     Only relevant if a maxValue has been defined. Default = false.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'rowConditionExpectation'
+               description: |
+                 Table rule which evaluates whether each row passes the specified condition.
+               properties:
+                 - !ruby/object:Api::Type::String
+                   name: 'sqlExpression'
+                   description: |
+                     The SQL expression.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'tableConditionExpectation'
+               description: |
+                 Table rule which evaluates whether the provided expression is true.
+               properties:
+                 - !ruby/object:Api::Type::String
+                   name: 'sqlExpression'
+                   description: |
+                     The SQL expression.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'dataProfileSpec'
+    description: |
+      DataProfileScan related setting.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'dataQualityResult'
+    output: true
+    description: |
+      The result of the data quality scan.
+    properties:
+      - !ruby/object:Api::Type::Boolean
+        name: 'passed'
```

Comment:
can we make the nested attributes here `output: true` as well please?

Reply:
+1